pytestqt/plugin.py

```diff
@@ -355,6 +355,10 @@ def __init__(self, timeout=1000, raising=False):
         self.timeout = timeout
         self.signal_triggered = False
         self.raising = raising
+        self._timer = QtCore.QTimer()
```

Comment:
Minor, but I think it would be cleaner to just create the timer if `timeout` is not `None`... then you can check for `self._timer is not None` during `_cleanup()`.